webapp/src/simulator.ts

```diff
@@ -307,6 +307,8 @@ export function run(pkg: pxt.MainPackage, debug: boolean,
         }
     }
 
+    const theme = pkg.config.theme || (pxt.appTarget.appTheme.matchWebUSBDeviceInSim && pxt.packetio.isConnected() && pxt.packetio.deviceVariant());
```

Comment:
I added this deviceVariant check in a previous pr and just noticed it was occasionally giving some spurious warnings in the console. very minor adjustment to check connected state here (basically no functional difference besides not logging when the result would be undefined anyways), didn't feel worth extracting from the webusb related pr :)